app/src/main/kotlin/com/wire/android/notification/WireFirebaseMessagingService.kt

```diff
@@ -22,13 +22,15 @@ class WireFirebaseMessagingService : FirebaseMessagingService() {
 
     override fun onNewToken(p0: String) {
         super.onNewToken(p0)
-        coreLogic.getAuthenticationScope().saveNotificationToken(p0, "GCM").let { result ->
+        coreLogic.globalScope {
+            saveNotificationToken(p0, "GCM")
```

Comment:
I see that is not your changes, but `p0` and hardcoded value..., I suggest to add some `todo`s to this part of the code.